tools/GenerateHelp.ps1

```diff
@@ -53,12 +54,16 @@ if ($ValidateMarkdownHelp)
         throw "No help folder found in the following services: $Services"
     }
 
-    $SuppressedExceptionsPath = "$PSScriptRoot\..\artifacts\Exceptions"
-    $NewExceptionsPath = "$PSScriptRoot\..\artifacts"
+    $NewExceptionsPath = "$PSScriptRoot\..\artifacts\StaticAnalysisResults"
+    if (!(Test-Path -Path $NewExceptionsPath))
+    {
+        New-Item -Path "$PSScriptRoot\..\artifacts" -Name "StaticAnalysisResults" -ItemType Directory
+    }
+    
     Copy-Item -Path "$PSScriptRoot\HelpGeneration\Exceptions\ValidateHelpIssues.csv" -Destination $SuppressedExceptionsPath
     New-Item -Path $NewExceptionsPath -Name ValidateHelpIssues.csv -ItemType File -Force | Out-Null
     Add-Content "$NewExceptionsPath\ValidateHelpIssues.csv" "Target,Description"
-    $FilteredHelpFolders | foreach { Validate-MarkdownHelp $_.FullName $SuppressedExceptionsPath $NewExceptionsPath }
+    $FilteredHelpFolders | foreach { Test-AzMarkdownHelp $_.FullName $SuppressedExceptionsPath $NewExceptionsPath }
```

Comment:
If we are goign to change these, then we shoudl change the help generation docs to use these new names as well